docker/build-release.sh

```diff
@@ -21,6 +21,27 @@ DOCKER_REPOSITORY=${DOCKER_REPOSITORY}
 #docker pull ${DOCKER_REPOSITORY}ubuntu:20.10
 
 # Build pravega-dev image which includes the source code and binaries for all applications.
```

Comment:
This comment refers to pravega-dev which is built below. Add a similar comment to describe the purpose of pravega-prod.